src/components/header/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import Search from "../search";
+import logo from "../../images/logo.svg";
+import upArrow from "../../images/upArrow.svg";
+import downArrow from "../../images/downArrow.svg";
+
+type HeaderItem = {
+  name: string;
+  subMenu: Array<HeaderItem>;
+  url: string;
+  ingress?: string;
+};
+
+type HeaderProps = {
+  headerItems: Array<HeaderItem>;
+};
+
+const Header = ({ headerItems }: HeaderProps) => {
+  const [dropDownOpen, setDropDownOpen] = useState(null);
```

Comment:
I would rename this to `openDropDown`, because `dropDownOpen` has a more booleanish feeling (at least in my mind).

And this should also be typed like: `useState<number | null>(null);`

Reply:
I think openDropDown sounds more like a task to function than the resulting state.

Reply:
Well yes and no, because of the word "open" can be either a verb or an adjective. How about `currentlyOpenDropDown`, although its a bit long? 

In any case `dropDownOpen` is misleading in my mind.